docs/src/pages/lab/breadcrumbs/breadcrumbs.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Breadcrumbs React component
+components: Breadcrumbs, Breadcrumb
+---
+
+# Breadcrumbs
+
+<p class="description">Breadcrumbs allow users to make selections from a range of values.</p>
+
+## Simple breadcrumbs
+
+{{"demo": "pages/lab/breadcrumbs/SimpleBreadcrumbs.js"}}
+
+## Custom separator
+
+The seperator can be any valid React node.
+In the following examples, we are using `"›"`, `"-"` and
```

Comment:
sorry to nitpick, but could we get a colon at the end, i.e. "and:"?

Alternatively, is this an example of "too much documentation"? Would "In the following examples, we are using two string separators, and an SVG icon." suffice?

Reply:
Are you changing it then? (The comment was marked as resolved, but the mardown is unchanged.